docs/4.x/media-analytics/options.md

```diff
@@ -192,7 +192,7 @@ _paq.push(['MediaAnalytics::scanForMedia']);
 Yes, the tracking of media events can be disabled by calling the following method:
 
 ```js
-_paq.push(['MediaAnalytics.disableTrackEvents']);
+_paq.push(['MediaAnalytics::disableTrackEvents']);
```

Comment:
maybe this was added by accident? It should only be a single dot AFAIK as the method exists on the tracker (not on `Matomo.MediaAnalytics`).

Reply:
yes its a mistake reverted back.
Thanks